lib/fastlane/plugin/wpmreleasetoolkit/helper/ios/ios_git_helper.rb

```diff
@@ -44,16 +44,22 @@ def self.localize_project()
           Fastlane::Helper::GitHelper.commit(message: "Update strings for localization", files: strings_files, push: true) || UI.message("No new strings, skipping commit")
         end
 
+        # Call the `Scripts/update-translations.rb` then the `fastlane/download_metadata` Scripts from the host project folder
+        #
+        # @env PROJECT_ROOT_FOLDER The path to the git root of the project
+        # @env PROJECT_NAME The name of the directory containing the project code (especially containing the `build.gradle` file)
+        #
         def self.update_metadata()
           Action.sh("cd #{ENV["PROJECT_ROOT_FOLDER"]} && ./Scripts/update-translations.rb")
-          Action.sh("git add #{ENV["PROJECT_ROOT_FOLDER"]}#{ENV["PROJECT_NAME"]}/*.lproj/*.strings")
-          Action.sh("git diff-index --quiet HEAD || git commit -m \"Updates translation\"")
 
-          Action.sh("cd fastlane && ./download_metadata.swift")
-          Action.sh("git add ./fastlane/metadata/")
-          Action.sh("git diff-index --quiet HEAD || git commit -m \"Updates metadata translation\"")
+          strings_files = Dir.chdir(File.join(ENV["PROJECT_ROOT_FOLDER"], ENV["PROJECT_NAME"])) do
+            Dir.glob("*.lproj/*.strings")
+          end
+          Fastlane::Helper::GitHelper.commit(message: "Update translations", files: strings_files, push: false)
 
-          Action.sh("git push origin HEAD")
+          Action.sh("cd fastlane && ./download_metadata.swift")
```

Comment:
This is a IMHO a high-value improvement opportunity – should we mark it with #todo or something? 😄

Reply:
Note that as said in the PR description:
 - I plan to at least extract that call to the script away from `ios_git_helper` in a subsequent PR (I actually hoped I'd be able to do that today but didn't have the time after all) – in which I will even try to completely get rid of those `[platform]_git_helper.rb` and the remaining things that I left there so far
 - I explicitly don't plan to make any changes in the way we do localization (i.e. via calling .swift or .sh scripts that are in project repos) during this Great Refactoring™ , as this will be part of a separate Project Thread that I plan to tackle after this one, but that will be a big project all on its own. That's why I plan to still keep the call to `./download_metadata.swift` so far and refactor this only in the next Project dedicated about Localization Tooling

Reply:
Oh for sure – I was thinking in the context of the project scope potentially shrinking that we may not get back to it any time soon, so it might be good to flag for future travellers, sorry!

Reply:
I might add a `@todo` indeed

Reply:
Fixed in e3264c721dfc9ec5ca78adbd3e08ec04c9c9f919